### PR TITLE
completed changes to teach tab and added scratchrj connect

### PR DESCRIPTION
**Pull request by epolak01**
### Resolves

_What Github issue does this resolve (please include link)? Please make sure that the PR title reflects what the PR will do, not what is changing.
Please do not submit PRs that are not associated with a Github issue, or that only partially implement an issue._

- Resolves #

### Proposed Changes

In the Teach tab, changes Curricula to Resources, and Assessments to ScratchJr Connect. In Resources,
adds introduction and changes the text at the top. In ScratchJr Connect, replaces previous content with 
a description of scratchjr connect website and links it with a button.

### Reason for Changes

To integrate scratchJr connect website.

### Test Coverage

_Please show how you have added tests to cover your changes, or provide the testing steps to verify the change. Screenshots that show the change are also helpful._

### CODE REVIEW

File: src/views/teach/connect.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Route, Switch} from 'react-router-dom';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import ConnectHomeSection from './connect/home.jsx';
+
+const ConnectSection = () => (
```
Comment (chrisgarrity) [inline src/views/teach/connect.jsx:7]: New connect stuff - yay

Comment (martinatan) [reply src/views/teach/connect.jsx:7]: Yes!

File: package.json [hunk 2]
```diff
@@ -20,7 +20,9 @@
     "url": "https://github.com/LLK/scratchjr-website/issues"
   },
   "homepage": "https://github.com/LLK/scratchjr-website",
-  "dependencies": {},
+  "dependencies": {
+    "lint": "^0.7.0"
```
Comment (chrisgarrity) [inline package.json:24]: Please remove this - eslint is already in the devDependencies, so this is unnecessary.

File: src/views/teach/activities/home.jsx [hunk 3]
```diff
@@ -16,17 +15,8 @@ const ActivitiesHome = () => (
             Each of these activities gives you a quick way to learn how
             to do new things with ScratchJr. They are listed here in
             order of simplest to hardest, but feel free to play around
-            in any order you&apos;d like! For more ScratchJr resources
-            developed by DevTech Research Group, click <a href="http://sites.tufts.edu/devtech/research/scratchjr/">here</a>.
+            in any order you&apos;d like!
```
Comment (chrisgarrity) [inline src/views/teach/activities/home.jsx:18]: Add 
For more information and a brief introduction to ScratchJr see the Resources page
```suggestion
            in any order you&apos;d like! For more information and a brief introduction to ScratchJr see the <a href=" /teach/resources">Resources page</a>.
```
I added a static link for resources page - I don't remember if that's the right way to do it when using react-router - please fix if needed.

**Review by chrisgarrity — CHANGES_REQUESTED** [review]
One small wording update. Otherwise looks great! thank you

**Comment by chrisgarrity** [comment]
@epolak01 @martinatan thank you!

**Pull request merged**